### PR TITLE
Fixed rules

### PR DESCRIPTION
**Pull request by darkdrgn2k**
Issue #220 

Corrections made
Removed duplication of ipv6 rules
Removed unused of ipv6 CJDNS target

### CODE REVIEW

**Comment by darkdrgn2k** [comment]
Note: Branched from master by accident, Travis is wrong on this one.

File: scripts/firewall/rules.v6 [hunk 1]
```diff
@@ -6,13 +6,6 @@
 -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 -A INPUT -p ipv6-icmp -j ACCEPT
 -A INPUT -i lo -j ACCEPT
--A INPUT -i tun0 -j cjdns
 -A INPUT -p tcp -m multiport --dports 80,443,9100,22,5201,4001
```
Comment (benhylau) [inline scripts/firewall/rules.v6]: Prometheus server and Grafana ports?

Comment (darkdrgn2k) [reply scripts/firewall/rules.v6]: C/P  issue #220

> Prometheus and Grafana are currently only available on ipv4 (which usually is the WiFi interface)

> Do we open then up in ipv6 (hyperboria) or ask user if they would like to do that?

I'll assume thats a yes?

Comment (benhylau) [reply scripts/firewall/rules.v6]: Why do we open some and not others? Where do we decide these are the ones we ask user about? If anything, it'd be node exporter that is sharing user information, not the prom server / grafana which just browse stuff right?

File: scripts/firewall/rules.v6 [hunk 2]
```diff
@@ -6,13 +6,6 @@
 -A INPUT -m conntrack --ctstate RELATED,ESTABLISHED -j ACCEPT
 -A INPUT -p ipv6-icmp -j ACCEPT
 -A INPUT -i lo -j ACCEPT
--A INPUT -i tun0 -j cjdns
--A INPUT -p tcp -m multiport --dports 80,443,9100,22,5201,4001
+-A INPUT -j ACCEPT -p tcp -m multiport --dports 80,443,9100,22,5201,4001
```
Comment (benhylau) [inline scripts/firewall/rules.v6]: Order the ports properly?

File: scripts/firewall/rules.v6 (hunk 1 above)
Comment (darkdrgn2k) [reply scripts/firewall/rules.v6]: Node Exporter - Point in time snapshot
Prometheus Server - Node Exporter data from any time that it was running
Grafana - Displays the above in a nice cpu intensive graph.

So one could argue that it gets progressively more invasive.  historical vs point in time.

Also grafana and prom are cpu/io hungry and cant really be used as a stack to monitor anything else but the local node really.

But I'm cool with opening those ports up if you still think it has value.

**Pull request merged**